dev/com.ibm.ws.cdi.extension.bce_fat/bnd.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2018, 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+# Define the bundle for this FAT
+
+src: \
+    fat/src,\
+    
+tested.features:
+    
+test.project: true
+
+-buildpath: \
+	io.openliberty.jakarta.cdi.4.0;version=latest,\
+	io.openliberty.jakarta.servlet.5.0;version=latest,\
+	com.ibm.websphere.javaee.annotation.1.2;version=latest,\
+	com.ibm.websphere.javaee.servlet.3.1
```

Comment:
Surely these should all be jakarta EE9/10 versions?
```suggestion
	io.openliberty.jakarta.cdi.4.0;version=latest,\
	io.openliberty.jakarta.servlet.5.0;version=latest,\
	io.openliberty.jakarta.annotation.2.0;version=latest
```

Reply:
I had problems with this as classes which use `FATServlet` wouldn't compile.

The Java EE versions of all of our test framework are automatically added to the classpath because it's a fat project and there's no way to remove them. If I manually add `com.ibm.ws.componenttest.2.0` then it does work with just the Jakarta versions of the API on the classpath.

I assume this means we end up with both versions of com.ibm.ws.componenttest on the classpath which isn't ideal but it does at least seem to compile (and at runtime it should be fine because we'll have the right version of the componenttest feature loaded).

Reply:
I think that manually adding `com.ibm.ws.componenttest.2.0` is the correct thing to do. I was assuming that the build scripts were smart enough not to add the other version as well.